src/System.Text.Encodings.Web/src/System/Text/Encodings/Web/TextEncoder.cs

```diff
@@ -132,9 +139,18 @@ public virtual string Encode(string value)
         // NOTE: The order of the parameters to this method is a work around for https://github.com/dotnet/corefx/issues/4455
         // and the underlying Mono bug: https://bugzilla.xamarin.com/show_bug.cgi?id=36052.
         // If changing the signature of this method, ensure this issue isn't regressing on Mono.
-        private unsafe int EncodeIntoBuffer(char* buffer, int bufferLength, char* value, int valueLength, int firstCharacterToEncode)
+        private unsafe OperationStatus EncodeIntoBuffer(
```

Comment:
Is the comment above this method still relevant? Seems like the issue has been fixed for a few years. If this comment isn't still relevant, please delete it.

Reply:
Sure. The issue was fixed in Mono 3.5 years ago, so deleting the comment.

However, I don't plan on changing the argument ordering since it was (source, destination):
`char* value, int valueLength, int firstCharacterToEncode, char* buffer, int bufferLength` and the current API is (destination, source) which is consistent with other similar APIs.